tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -199,3 +210,89 @@ async def test_restore_state(mock_heat_meter, hass: HomeAssistant) -> None:
     assert state
     assert state.state == "devicenr_789"
     assert state.attributes.get(ATTR_STATE_CLASS) is None
+
+
+@patch(API_HEAT_METER_SERVICE)
+async def test_exception_during_setup(mock_heat_meter, hass: HomeAssistant) -> None:
```

Comment:
I don't think this test is correct.
You are setting up the mock, but then resetting it straight away and finally only asserting that the mock was created ?!?
Or maybe I am just not understanding this test?